doc/nginx-default.conf

```diff
@@ -5,10 +5,12 @@ server {
 
     #access_log  /var/log/nginx/host.access.log  main;
 
+    absolute_redirect off;
+    rewrite ^/$ /MLOS/ redirect;
```

Comment:
Convenience to redirect from `http://localhost:8080` to `http://localhost:8080/MLOS` but only for the top level path.